code/DebugBarDatabaseCollector.php

```diff
@@ -43,9 +43,9 @@ public function collect()
             DebugBar::getDebugBar()
                 ->getCollector('messages')
                 ->addMessage(
-                    'Your page is running a high number of database queries. You could reduce this by implementing '
-                    . 'caching. <a href="https://docs.silverstripe.org/en/developer_guides/performance/"'
-                    . ' target="_blank">More information.</a>',
+                    'This page ran more than ' . $dbQueryWarningLevel . ' database queries. You could reduce this by '
```

Comment:
@robbieaverill Could we make this URL configurable? It'd be good to point CWP devs to the more platform specific perf guide at https://www.cwp.govt.nz/developer-docs/en/performance_guide/